From 47f4c00adaa442cd03b02d44517b5503718f2813 Mon Sep 17 00:00:00 2001
From: Christophe Giboudeaux <christophe@krop.fr>
Date: Tue, 12 Jun 2018 22:52:06 +0200
Subject: Fix build with Qt 5.11.

Summary: 'id' is an int, it doesn't need qPrintable.

Reviewers: johanneszarl

Differential Revision: https://phabricator.kde.org/D13329
---
 RemoteControl/RemoteConnection.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/RemoteControl/RemoteConnection.cpp b/RemoteControl/RemoteConnection.cpp
index 1dacb72..8bc18d0 100644
--- a/RemoteControl/RemoteConnection.cpp
+++ b/RemoteControl/RemoteConnection.cpp
@@ -105,7 +105,7 @@ void RemoteConnection::dataReceived()
             std::unique_ptr<RemoteCommand> command = RemoteCommand::create(static_cast<CommandType>(id));
             command->decode(stream);
             protocolDebug() << qPrintable(QTime::currentTime().toString(QString::fromUtf8("hh:mm:ss.zzz")))
-                               << ": Received " << qPrintable(id);
+                               << ": Received " << id;
 
             emit gotCommand(*command);
         }
-- 
cgit v0.11.2
